### DISCUSSION TITLE
Custom template to provide at time of code generation to add more tags to the generated golang structs.

### DISCUSSION THREAD

**Discussion by MUzairS15** [oapi-codegen/oapi-codegen#1686] [Q&A] [answered]
How do we provide a custom template to have `json` and `yaml` tags to the generated structs?
I am aware that I can use `x-oapi-codegen-extra-tags` to add custom tags but it is tedious to add for each schema type.

**Comment by jamietanna** [comment, marked as answer]
Closed by https://github.com/oapi-codegen/oapi-codegen/issues/1696
